docs/index.rst

```diff
@@ -0,0 +1,7 @@
+
+Welcome to Python Fluent's documentation!
+=========================================
+
+The :py:mod:`python-fluent` library contains a
+`syntax <fluent.syntax/stable/>`_ and a
+`runtime <fluent.runtime/stable/>`_ library.
```

Comment:
This single sentence is short and sweet, but also almost easy to miss entirely. Would mind adding a short intro paragraph, similar to the one in the top-level README, and listing each package in a bullet point?

Reply:
Also, is there any way to add links to each package's docs in the sidebar?

Reply:
Figured out a way, but it's hacky. The "right" way is probably to set up intersphinx, but I want to start dealing with that once we have docs up on projectfluent, so that dependencies could resolve at least in theory.